train/train.py

```diff
@@ -153,7 +153,7 @@ def train_step(batch, adver_batch, step_i):
             discriminator_loss += calc_discriminator_loss(discrim_on_real, discrim_on_pred)
             generator_loss = loss_fn(tf.ones_like(discrim_on_pred), discrim_on_pred)
             gen_acc_metric.update_state(tf.ones_like(discrim_on_pred), discrim_on_pred)
-            loss += generator_loss * FLAGS.adver_weight
+            loss += generator_loss * float(step_i / max_train_steps) * FLAGS.adver_weight
```

Comment:
is the intuition here to solve the task first, then worry about making convincing states later?

Reply:
Yeah, good summary.